validator/coordinator/src/test/java/tech/pegasys/teku/validator/coordinator/ValidatorApiHandlerTest.java

```diff
@@ -519,6 +521,18 @@ public void subscribeToBeaconCommittee_shouldUpdateActiveValidatorTrackerWhenNot
     verify(activeValidatorTracker).onCommitteeSubscriptionRequest(validatorIndex, aggregationSlot);
   }
 
+  @Test
+  void subscribeToSyncCommitteeSubnets_shouldConvertCommitteeIndexToSubnetId() {
+    final SyncCommitteeSubnetSubscription subscription1 =
+        new SyncCommitteeSubnetSubscription(1, Set.of(1, 2, 15, 30), UInt64.valueOf(44));
+    final SyncCommitteeSubnetSubscription subscription2 =
+        new SyncCommitteeSubnetSubscription(1, Set.of(5, 10), UInt64.valueOf(35));
+    validatorApiHandler.subscribeToSyncCommitteeSubnets(List.of(subscription1, subscription2));
+    System.out.println(spec.getSyncCommitteeUtilRequired(ZERO).getSubcommitteeSize());
+    verify(syncCommitteeSubscriptionManager).subscribe(0, subscription1.getUntilEpoch());
+    verify(syncCommitteeSubscriptionManager).subscribe(0, subscription1.getUntilEpoch());
```

Comment:
so does this convert to subnet 0 calls the subscription manager 4 times and subnet1 calls it 2 times?
can probably use `verify(syncCommitteeSubscriptionManager, times(4))`
and second call should be checking subscription 2 i think?

Reply:
Ah yes, that test got far enough to reveal the problem but didn't actually get finished to be completely explicit about what we are expecting.